docs/migrating.rst

```diff
@@ -0,0 +1,895 @@
+    *   - ``session``
+        - The session parameter is a new parameter that allows you to pass a :class:`aiohttp.ClientSession` to the client. By default this is an optional parameter. This parameter comes in useful if you want to manage a session yourself. In that case, it is a required parameter.
+    *   - ``beta``
+        - Denotes if the client has the permissions to access beta endpoints (always prefixed with ``beta_fetch_x``). This is a new parameter that is set to ``False`` by default. If you want to access beta endpoints, you must set this parameter to ``True``.
+    *   - ``optimization_flags``
+        - This is a new parameter that allows you to toggle certain optimizations on or off. This parameter is not required, and defaults to :meth:`~fortnite_api.OptimizationFlags.default` if not given. 
+
+See the documentation for these parameters in :class:`fortnite_api.Client`.
+
+Sync client
+^^^^^^^^^^^
+.. list-table::
+    :header-rows: 1
+
+    *   - Parameter
+        - Description
+    *   - ``api_key``
+        - The API key to use for requests to the Fortnite API. This is not a required parameter for most endpoints however, it is required to use any stats endpoints. This is the same as in Version 2. 
+    *   - ``default_language``
+        - The client now boasts a default language parameter. For many endpoints in the Fortnite API, you can specify a language parameter to denote the language you want the response to be in. This parameter allows you to set a default language for all requests made by the client, for endpoints that support it, so that you don't have to specify it every time you make a request. The default language is :attr:`~fortnite_api.GameLanguage.ENGLISH` (``en``). 
+    *   - ``session``
+        - The session parameter is a new parameter that allows you to pass a :class:`requests.Session` to the client. By default this is an optional parameter. This parameter comes in useful if you want to manage a session yourself. In that case, it is a required parameter.
+    *   - ``beta``
+        - Denotes if the client has the permissions to access beta endpoints (always prefixed with ``beta_fetch_x``). This is a new parameter that is set to ``False`` by default. If you want to access beta endpoints, you must set this parameter to ``True``.
+    *   - ``optimization_flags``
+        - This is a new parameter that allows you to toggle certain optimizations on or off. This parameter is not required, and defaults to :meth:`~fortnite_api.OptimizationFlags.default` if not given. 
+
+See the documentation for these parameters in :class:`fortnite_api.SyncClient`.
```

Comment:
This specific design choice was made to link new users to the migration documentation. It may be unnecessarily difficult for users to understand the migration guide if they are trying to determine which client they are using and which parameters line up with which client... I feel it's going to lead to silly questions. This clears up any ambiguity - What are your thoughts?

Reply:
I thought about starting that both clients params are identical. Only session is a different type, that can be mentioned in the row by splitting it into 2 parts.